### PR TITLE
Portfolio : Details popup-window

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
We did the following;

-     When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
-     When the user clicks (or taps) the close (X) button, the popup disappears.
- Created the work section dynamically in javascript.
- Each project stores the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, and link to the source.

Below is a link to my partner's contribution to the project
https://github.com/arnoldnekemiah/portfolio-new/commit/d503affcf2a355a0c22f58b11ccd6b7286f82126

Thank you :smiley:

### CODE REVIEW

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @arnoldnekemiah & @marvin-nyalik, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good implementing the popup windows, it's look good on the desktop version :1st_place_medal: 
- Good commit :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [x] Do not commit commented out code in your code base. They will make your project look messy. Kindly remove all commented code from your project (e.g. `index.html from line 107 to 222`)

- [x] (`main.js from line 41 to line 172`) In the project requirement, it requested to create dynamically the project from the array containing all the project, but in your code the projects are created manually. Kindly loop through the array containing the projects to display them dynamically.

- [x] Kindly make sure the popup window is populated with the information of its project, instead of populating the same information for all the project.

- [x] When clicking the button to check project details for the first project ("My Recent Works"), its popup windows doesn't appears. Kindly check it to show your ability to use JavaScript to manipulate the DOM elements.

- [x] Nice work implementing the popup windows for desktop version :clap: However it not showing well on the mobile version. Kindly make sure your popup windows it showing well on both mobile and desktop version to follow the Figma design.
### Your design :point_down: 
Screenshot from 2023-05-18 20-20-37
### What it is supposed to look like :point_down: 
Screenshot from 2023-04-18 17-53-42

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 1]
```diff
@@ -37,198 +37,187 @@ function checkWindowSize() {
...
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
+  {
+    heading: 'Profesional Art Printing Data',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard',
+    technologies: ['html', 'CSS', 'bootstrap', 'Ruby'],
+  },
```
Comment (V-Blaze) [inline main.js]: - [ ] You have done well so far, :+1: :100: But I think there are still some requirements you need to meet before moving to the next milestone.
- - As the previous reviewer suggested, You are meant to have one array of projects where you loop through and display projects on the page and also show more details with the pop-up when the button is clicked, currently, you tend to have two arrays, One for projects that display on the homepage and one for the pop-up, That should not be the case, I'll suggest you delete the `projects` array and then loop through `data` array to display them on the homepage, and also bring the **popup to show more details of that given project when the `see project` button is clicked** :smile: :wave:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @arnoldnekemiah , 

# Almost there 🟡 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, and some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Pop up looks great✔
- Descriptive pull request title :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @arnoldnekemiah 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Great job completing the requested changes from the previous reviewers

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @arnoldnekemiah👋,

_This is an additional review as a result of the second opinion request._

I can see that your project has been approved. Good luck on your upcoming projects 🚢 

Keep rocking 🚀!

**Pull request merged**